### PR TITLE
Fix/theme for accordion

### PR DESCRIPTION
**Pull request by adel0896**
## This PR

Closes #65184

## Notes

- No notes to add

### Fulfilled the scope of this repo?

- [ ] The component implements an element of the [DHI Design Guidelines](https://www.figma.com/file/pSfX5GNsa6xhKGbi3DWQtn/DHI-Official-Guidelines) , the Accordion and it modifies the style of it

### Completness

Skærmbillede 2023-03-17 130059
Skærmbillede 2023-03-21 190010
Skærmbillede 2023-03-21 190033

### CODE REVIEW

**Review by bertearazvan — COMMENTED** [review]
I admire you looked into how the MuI classes are handled, for me it was a headache for a long while until I got the gist of it. So, very nice job!!

Before approving I would like to challenge the solution a bit. 

My reasoning is: _We should rely as much as possible on the default mui way of handling colors, and transitions_. 

If in the design systems the UI looks in a way, but mui by default handles it in another (like the pointerEvent, transition), we should aim to rely on it so that we bring as little customization as possible.

File: packages/react-components-lab/src/components/ThemeProvider/getDhiOverrides.ts [hunk 1]
```diff
@@ -340,15 +340,28 @@ const getDhiOverrides = (mode: PaletteMode): Components => {
...
+            PointerEvent: 'none',
+            opacity: 0.4,
+          },
+          '& .MuiAccordionSummary-root': {
+            '& .MuiTypography-root': {
+              fontWeight: 700,
+            },
+          },
+          '& .MuiAccordionSummary-expandIconWrapper .MuiSvgIcon-root': {
+            fill: dhiPalette.secondary.dark,
```
[the remark this answers is not in the document]
Comment (adel0896) [reply packages/react-components-lab/src/components/ThemeProvider/getDhiOverrides.ts:357]: Here the icon directly from MUI was a type of grey, so it was not matching the text

File: packages/react-components-lab/src/components/ThemeProvider/getDhiOverrides.ts [hunk 2]
```diff
@@ -340,15 +340,28 @@ const getDhiOverrides = (mode: PaletteMode): Components => {
     MuiAccordion: {
       styleOverrides: {
         root: {
-          '& .Mui-expanded': {
+          '& .MuiAccordionSummary-root.Mui-expanded': {
             backgroundColor: dhiPalette.grey[50],
+            transition: 'none',
```
[the remark this answers is not in the document]
Comment (adel0896) [reply packages/react-components-lab/src/components/ThemeProvider/getDhiOverrides.ts:345]: I have to actually check if that is still needed, you are right. I think I made a few more changes than necessary while experimenting with it, as there were some weird background colors changing when we were clicking on one summary while it also had a hover effect. I will take another look a it here though, and also implement one more hover effect Mirela mentioned when we had the discussion

File: packages/react-components-lab/src/components/ThemeProvider/getDhiOverrides.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (bertearazvan) [reply packages/react-components-lab/src/components/ThemeProvider/getDhiOverrides.ts:357]: Alright, that sounds fair 💯 Good decision!

**Review by bertearazvan — APPROVED** [review]
🚀

Remember to update the version, push, build and publish

**Pull request merged**